bundles/http/Citrix/AdobeSign/31fff7a5-ffa8-4019-9e2f-e477837f92de.js

```diff
@@ -0,0 +1,596 @@
+    const userids = await syncUsers(dataStore, client, integrationParameters,);
+    const agreements = await syncAgreements(dataStore, client, userids, integrationParameters, latestSynchronizationTime);
+    await syncArgreementDetails(client, dataStore, agreements)
+}
+
+async function libraryDocumentView({ client, dataStore, actionParameters }) {
+    const response = await client.fetch(`/api/rest/v6/libraryDocuments/${actionParameters.libraryDocumentId}/views`, {
+        method: 'POST',
+        headers: {
+            "x-api-user": `email:${actionParameters.email}`,
+        },
+        body: JSON.stringify({
+            name: actionParameters.name,
+            commonViewConfiguration: {
+                autoLoginUser: true
+            }
+        })
+    })
+    const json = await response.json()
+    if (!response.ok) {
+        throw new Error(JSON.stringify(json))
+    }
+    const libraryViewList = json['libraryViewList'] ?? []
+    let data = libraryViewList.map(libraryView => {
+        return {
+            "libraryDocumentId": actionParameters.libraryDocumentId,
+            "email": actionParameters.email,
+            "url": libraryView.url,
+            "embeddedCode": libraryView.embeddedCode,
+            "name": libraryView.name,
+            "isCurrent": libraryView.isCurrent
+        }
+    })
+    dataStore.save("libraryDocumentViews", data)
+}
+
+async function agreementview({ client, dataStore, actionParameters }) {
+    const response = await client.fetch(`/api/rest/v6/agreements/${actionParameters.agreementId}/views`, {
+        method: 'POST',
+        headers: {
+            "x-api-user": `email:${actionParameters.email}`,
+        },
+        body: JSON.stringify({
+            name: actionParameters.name,
+            commonViewConfiguration: {
+                autoLoginUser: true
+            }
+        })
+    })
+    const json = await response.json()
+    if (!response.ok) {
+        throw new Error(JSON.stringify(json))
+    }
+    const agreementViewList = json['agreementViewList'] ?? []
+    let data = agreementViewList.map(agreementView => {
+        return {
+            "agreement_id": actionParameters.agreementId,
+            "email": actionParameters.email,
+            "url": agreementView.url,
+            "embeddedCode": agreementView.embeddedCode,
+            "expiration": agreementView.expiration,
+            "name": agreementView.name,
+            "isCurrent": agreementView.isCurrent
+        }
+    })
+    dataStore.save("agreementViews", data)
+}
+
+async function sendAgreement({ dataStore, client, actionParameters, integrationParameters }) {
+    let transientDocuments = null
+    let fileInfos = {
+        "libraryDocumentId": `${actionParameters.libraryDocumentId}`
+    }
+    if (actionParameters.libraryDocumentId == null) {
+        let transientDocument = await uploadFiles(dataStore, client, actionParameters)
+        if(transientDocument == false){
```

Comment:
From `uploadFiles()` you return `json.transientDocumentId`
It looks like number, so it can be `0`.
(yeah, I know that id probably never be 0, but who knows)

Then it's very simple to use just `===` and you don't have to care

```suggestion
        if (transientDocument === false){
```

Reply:
here from `uploadFiles()`  we will return 0 instead of false, and in `sendAgreement()` we will typecast it to Boolean and then we will compare ` if(Boolean(transientDocument) === false){
            throw new Error('File not selected')
        }`